NthKey.xcodeproj/project.pbxproj

```diff
@@ -25,7 +25,7 @@
 		2AEDBE2C25FB3F0B0018C5CF /* AddressesViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AEDBE2B25FB3F0B0018C5CF /* AddressesViewModel.swift */; };
 		2AEECEEE262C6A4100578819 /* String.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AEECEED262C6A4000578819 /* String.swift */; };
 		2AF86BCF2608B18000A770AD /* DataManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF86BCE2608B18000A770AD /* DataManager.swift */; };
-		2AF86C34260B555C00A770AD /* Network.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF86C33260B555B00A770AD /* Network.swift */; };
+		2AF86C34260B555C00A770AD /* NetworkOfWallet.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2AF86C33260B555B00A770AD /* NetworkOfWallet.swift */; };
```

Comment:
Maybe `WalletNetwork`?

Reply:
fixed in 54c4d9d7cdfd54ce303c33ef566a40af85dd2395